Lusas_oM/Versioning_70.json

```diff
@@ -0,0 +1,25 @@
+{
+  "Namespace": {
+    "ToNew": {
+    },
+    "ToOld": {
+    }
+  },
+  "Type": {
+    "ToNew": {
+    },
+    "ToOld": {
+    }
+  },
+  "Property": {
+    "ToNew": {
+    },
+    "ToOld": {
+    }
+  },
+  "MessageForDeleted": {
+    "BH.oM.Adapters.Lusas.LusasConfig": "This object has been replaced by the LusasSettings object. For further assistance, please raise an issue on https://github.com/BHoM/Lusas_Toolkit/issues."
```

Comment:
Can we direct the users to this PR in particular.

Reply:
We can for sure. Changed in the latest commit.